webapp/src/main/java/io/github/microcks/util/delay/FixedDelayStrategy.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The Microcks Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.microcks.util.delay;
+
+/**
+ * A {@link DelayStrategy} that returns fixed value.
+ * <p>
+ * Instances should be obtained via the factory method in {@link DelayStrategyFactory} to ensure proper usage and
+ * encapsulation.
+ * </p>
+ * @author SebastienDegodez
+ */
+public class FixedDelayStrategy implements DelayStrategy {
+   FixedDelayStrategy() {
+   } // package-private
+
+   @Override
+   public long compute(long baseValue) {
+      return baseValue;
+   }
+
+   @Override
+   public String getName() {
+      return "fixed";
```

Comment:
We need to extract this to a constant here. Will be typically useful when creating a default delay with non specified strategy.